pytorch_lightning/accelerators/gpu.py

```diff
@@ -45,6 +45,8 @@ def setup(self, trainer: "pl.Trainer") -> None:
         self.set_nvidia_flags(trainer.local_rank)
         # clear cache before training
         torch.cuda.empty_cache()
+        # benchmarking
+        torch.backends.cudnn.benchmark = trainer.benchmark
```

Comment:
I'm not sure it matters, but the accelerator.setup() get called way later than accelerator_connector.init, will move to setup_envirment() make more sense?

Reply:
I'm not sure how much of a difference that will make, it looks like they get called one after the other: https://github.com/PyTorchLightning/pytorch-lightning/blob/9826de21625b162dfef43f30864b5a6d9efab003/pytorch_lightning/trainer/trainer.py#L1106-L1107

I'm guessing that as long as it is set before training it should be fine, but someone please correct me if I'm wrong.